script/test_Dockerfile

```diff
@@ -16,16 +16,24 @@ RUN apt-get update && apt-get install -y \
     python-minimal \
     --no-install-recommends
 
+# install bats 
+RUN cd /tmp \
```

Comment:
kk... np... updated to reset head back to a known commit (which is the latest atm).

Cheers.